### ISSUE TITLE
Provide Python packages on pypi

### ISSUE THREAD

**Issue by bekaus** [ukoethe/vigra#106]
There should be a python-vigra package on http://pypi.python.org.

The reasons:
- quick install on new machines with easy_install resp. pip
- proper dependency handling in setup.py files for code depending on vigra-numpy
- setting up an automatic build system for code depending on vigra-numpy

**Comment by emmenlau** [comment]
Yes that would be great. At least I've searched now for a while where the python bindings are, since several distributions name them "python-vigra". It was rather confusing not to find anything related on pypi. It adds to the confusion that a pyVIGRA package exists.
